### ISSUE TITLE
No Prop Hints using index.es.js

### ISSUE THREAD

**Issue by kajchang** [transitive-bullshit/create-react-library#157]
When I use the typescript starter and I import it, the imported components don't have typehints for the props. They only have typehints if I import them from `react-library-name/dist/index`.

**Comment by kajchang** [comment]
this is on webstorm

**Comment by kajchang** [comment]
I found this library: https://github.com/joelday/ts-proptypes-transformer
I'm trying to use it as a transformer with the typescript plugin:
```js
import proptypes from 'ts-proptypes-transformer'

typescript({
  rollupCommonJSResolveHack: true,
  clean: true,
  transformers: [service => ({
    before: [],
    after: [proptypes(service.getProgram())]
  })]
}),
```
I know this is pretty out of scope for this project, but I wonder if anyone knows how to fix the error I get: 
```
[!] (rpt2 plugin) TypeError: Cannot read property 'kind' of undefined
src/index.tsx
TypeError: Cannot read property 'kind' of undefined
```

**Comment by transitive-bullshit** [comment]
This should be fixed now with the latest `v3` release.

Please open a new issue if you're still seeing errors.
